lib/api/apiUtils/object/locationStorageCheck.js

```diff
@@ -0,0 +1,54 @@
+const { errors } = require('arsenal');
+
+const { config } = require('../../../Config');
+const { listLocationMetric, pushLocationMetric } =
+    require('../../../utapi/utilities');
+
+function _gbToBytes(gb) {
+    return gb * 1024 * 1024 * 1024;
+}
+
+/**
+ *
```

Comment:
May be worth mentioning in the JSDoc that it might not enforce the limit exactly in case there are concurrent requests coming while reaching the storage limit (atomicity issue between `listLocationMetric` and `pushLocationMetric`)